src/frontend/apps/impress/src/features/docs/doc-export/components/ModalExport.tsx

```diff
@@ -98,9 +100,12 @@ export const ModalExport = ({ onClose, doc }: ModalExportProps) => {
         exportDocument,
       )) as React.ReactElement<DocumentProps>;
 
-      // Inject language for screen reader support
+      // Inject language and title for screen reader support and accessibility
       const pdfDocument = isValidElement(rawPdfDocument)
-        ? cloneElement(rawPdfDocument, { language: i18next.language })
+        ? cloneElement(rawPdfDocument, {
+            language: i18next.language,
+            title: documentTitle,
```

Comment:
I can see we get the information now in the test e2e:
```ts
console.log(pdfData.info);
 {
      PDFFormatVersion: '1.3',
      IsAcroFormPresent: false,
      IsXFAPresent: false,
      Title: 'chromium-3797-0-doc-editor-line-break',
      Creator: 'react-pdf',
      Producer: 'react-pdf',
      CreationDate: 'D:20250917092655Z'
  }
```

Could you add a check in one of the test to assert that the `Title` is set ?
Do not create a special test for that, just update one of them, tests e2e are quite heavy for the CI.